configure.ac

```diff
@@ -8,7 +8,7 @@ AM_CONFIG_HEADER(config.h)
 AM_INIT_AUTOMAKE([subdir-objects])
 
 AC_SUBST(RELEASE, ${RELEASE:-unknown})
-AC_SUBST(TARBALL, ${TARBALL:-${PACKAGE}-${VERSION}.tar.gz})
+AC_SUBST(TARBALL, ${TARBALL:-${VERSION}.zip})
```

Comment:
Why remove ${PACKAGE}?

Right now changing this breaks being able to build directly with a make dist command.

I'm not familiar with autoconf enough to know if make dist can generated a zip file?

Would you happen to know?

Reply:
> Right now changing this breaks being able to build directly with a make dist command.

Reply:
Sorry I was not clear.  Make dist is fine but it outputs a tar.gz not a zip file.  So the spec file within the tar ball can't find the zip file and I get the following error when I run:

./autogen.sh && ./configure && make dist && rpmbuild -ta infiniband-diags-2.1.0.tar.gz
...
error: Bad source: /home/iweiny/dev/management/infiniband-diags/2.1.0.zip: No such file or directory

Basically the output from make dist is no longer able to be used to build with rpmbuild...  Perhaps that is old school and we should not care?

I'm not familiar with spectool but it looks like you need to have access to github for it to run?  I can probably test that but not sure I will have time today.

Reply:
Thank you for explaining your workflow.
I reverted the definition of the `TARBALL` macro in `configure.ac` to its original definition and modified the `Source` in the spec.in file to download the tar.gz file from github